docs/getting-started-wsl.md

```diff
@@ -0,0 +1,49 @@
+# Software Setup on Windows Subsystem for Linux
+
+## Table of Contents
+
+* [Table Of Contents](#table-of-contents)
+* [Introduction](#introduction)
+* [WSL2 Setup](#wsl2-setup)
+* [X Server Setup](#wsl2-setup)
+
+
+## Introduction
+
+Windows has a Windows Subsystem for Linux component that can be used to develop and run code for Linux on Windows. WSL1 was a Windows component that implemented Linux kernel interfaces, and didn't work great with Thunderbots AI software or grSim. WSL2 runs a full-fledged Linux kernel in a VM, and works great with Thunderbots AI software and grSim with the exception that we need to use software rendering instead of GPU-accelerated rendering for grSim and our AI.
+
+Note that this will not work with legacy robots. Due to the lack of USB support in WSL2, we are unable to use the USB dongle used to communicate with them.
```

Comment:
Might be good to put the note at the top so ppl can decide whether this option is right for them

Reply:
updated with some disclaimers